common/focuses/wc_magic_lifestyle_focuses.txt

```diff
@@ -0,0 +1,927 @@
+﻿light_magic_focus_1 = {
+	lifestyle = light_magic_lifestyle
+
+	is_valid = {
+		MAGIC_lifestyle_is_valid_trigger = { MAGIC = light }
```

Comment:
So the player and AI can keep developing the banned magic if religious laws change?

Reply:
Player and AI can be witches and cannibals while following faiths that criminalise it?

Reply:
> Player and AI can be witches and cannibals while following faiths that criminalise it?

That's another question, but I hope you will come with a solution later.
So far, you can't pick the criminal lifestyles at all. Unless you started it before it was criminalized.

Reply:
> So far, you can't pick the criminal lifestyles at all. Unless you started it before it was criminalized.

You just need exp in that lifestyle, so it can be allowed to use by an event/decision/interaction/whatever that gives exp/perk in criminalised lifestyle.

Reply:
> You just need exp in that lifestyle, so it can be allowed to use by an event/decision/interaction/whatever that gives exp/perk in criminalised lifestyle.

I see.

> AI can be witches and cannibals

So, zealous Gul'dan with the Holy Light religion will keep developing Fel magic? I think you should set some kind of block like having Cynical trait (thus the AI ignore laws).

Reply:
> I think you should set some kind of block like having Cynical trait (thus the AI are likely to ignore society laws).

True

Reply:
Done

Reply:
> 
> 
> > I think you should set some kind of block like having Cynical trait (thus the AI are likely to ignore society laws).
> 
> True

Suggestions for other traits that should allow it: Possessed, Lunatic, and maybe Ambitious.